### PR TITLE
Docs and package updates

### PR DESCRIPTION
**Pull request by britt-allen**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ nav:
   - Setup: setup.md
   - Writing Documentation: writing-documentation.md
   - Data Warehouse Performance: data-warehouse-performance.md
-  - dbt Project: /dbt_docs/index.html
+  - dbt Project: ../dbt_docs/index.html
```
Comment (ian-r-rose) [inline mkdocs.yml:29]: This might need to be `./` rather than `../`. I suggest trying to build the docs from this branch to test it out

Comment (ian-r-rose) [reply mkdocs.yml:29]: That is to say, temporarily add this branch name to here: https://github.com/cagov/caldata-mdsa-caltrans-pems/blob/56eb01a9cae7d92ddfaffa594288698635fcb231/.github/workflows/docs.yml#L5

Comment (britt-allen) [reply mkdocs.yml:29]: I forgot how to do that lol, I'll read the docs. Thank you!

Comment (ian-r-rose) [reply mkdocs.yml:29]: (`../` might also be correct, I'm not sure off the top of my head)

Comment (britt-allen) [reply mkdocs.yml:29]: Found it: https://cagov.github.io/caldata-mdsa-caltrans-pems/writing-documentation/

File: mkdocs.yml [hunk 2]
```diff
@@ -26,4 +26,4 @@ nav:
   - Setup: setup.md
   - Writing Documentation: writing-documentation.md
   - Data Warehouse Performance: data-warehouse-performance.md
-  - dbt Project: /dbt_docs/index.html
+  - dbt Project: dbt_docs_snowflake/index.html
```
Comment (ian-r-rose) [inline mkdocs.yml:29]:
```suggestion
  - dbt Project: dbt_docs/index.html
```

Comment (britt-allen) [reply mkdocs.yml:29]: Did this locally before seeing the suggestion here, thanks!

Comment (ian-r-rose) [reply mkdocs.yml:29]: I don't see this change in your branch

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - docs_and_package_updates
```
Comment (ian-r-rose) [inline .github/workflows/docs.yml:6]: Docs build looks fixed now, let's remove this and merge!

Comment (britt-allen) [reply .github/workflows/docs.yml:6]: Oh duh, ty!

**Pull request merged**